packages/consent-manager/style.module.css

```diff
@@ -0,0 +1,5 @@
+.root {
+  @media only percy {
```

Comment:
Closes out [this task](https://app.asana.com/0/1100423001970639/1200759443625270/f) and does the half the required work for [this task](https://app.asana.com/0/1100423001970639/1200916857009754/f).